genesyscloud/resource_genesyscloud_outbound_callabletimeset.go

```diff
@@ -0,0 +1,320 @@
+package genesyscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v75/platformclientv2"
+	"github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/consistency_checker"
+	"log"
+	"time"
+)
+
+var (
+	outboundcallabletimesetcallabletimeResource = &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			`time_slots`: {
+				Description: `The time intervals for which it is acceptable to place outbound calls.`,
+				Required:    true,
+				Type:        schema.TypeSet,
+				Elem:        outboundcallabletimesetcampaigntimeslotResource,
+			},
+			`time_zone_id`: {
+				Description: `The time zone for the time slots; for example, Africa/Abidjan`,
+				Required:    true,
+				Type:        schema.TypeString,
+			},
+		},
+	}
+	outboundcallabletimesetcampaigntimeslotResource = &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			`start_time`: {
+				Description: `The start time of the interval as an ISO-8601 string, i.e. HH:mm:ss`,
+				Required:    true,
+				Type:        schema.TypeString,
+			},
+			`stop_time`: {
+				Description: `The end time of the interval as an ISO-8601 string, i.e. HH:mm:ss`,
+				Required:    true,
+				Type:        schema.TypeString,
+			},
+			`day`: {
+				Description: `The day of the interval. Valid values: [1-7], representing Monday through Sunday`,
+				Required:    true,
```

Comment:
You could add a `ValidateFunc: validation.IntInSlice(),` here to check if it's 1,2,3 etc